common/persistence/persistence-tests/persistenceTestBase.go

```diff
@@ -67,6 +67,10 @@ import (
 	"go.temporal.io/server/service/history/tasks"
 )
 
+// TimePrecision is needed to account for database timestamp precision.
+// Cassandra only provides milliseconds timestamp precision, so we need to use tolerance when doing comparison
+const TimePrecision = 2 * time.Millisecond
```

Comment:
this is not used anywhere in this PR?

Reply:
it is used by existing persistence tests, simply moving the definition from file to be deleted to here